.github/workflows/release_npm_version.yml

```diff
@@ -28,9 +29,15 @@ jobs:
       # Setup .npmrc file to publish to npm
       - uses: actions/setup-node@v3
         with:
-          node-version: '16.x'
+          node-version: '18.x'
           registry-url: 'https://registry.npmjs.org'
 
+      - name: Install dependencies
+        run: |
+          rm -rf node-modules
+          rm package-lock.json
+          npm install
+
```

Comment:
Won't this result in a difference in our dependency versions between what is published to NPM and the GH tag?

Reply:
Not sure what you mean. This is just installing the same dependencies in the package.json file that the next step (npm publish tag) will use.

Reply:
updating to using yarn for installing dependencies.